fastcrypto-zkp/src/bn254/zklogin_test_vectors.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
As discussed, we need to add JWKs corresponding to these JWTs in https://github.com/MystenLabs/zklogin-circuits/blob/main/src/pastJWKs.ts.